api/catalog/api/serializers/audio_serializers.py

```diff
@@ -62,35 +64,23 @@ class AudioSearchRequestSerializer(
 
     category = serializers.CharField(
         label="category",
-        help_text="A comma separated list of categories; available categories "
-        "include `music`, `sound_effect`, `podcast`, `audiobook`, "
-        "and `news`.",
+        help_text=make_comma_separated_help_text(AUDIO_CATEGORIES, "categories"),
```

Comment:
This is fantastic, but a small part of me wants to extend `CharField`, give it a keyword argument (like `options` or `values`) and in `__init__`, generate this help text automatically. Again, this is clear and excellent and definitely a huge improvement **to merge as is**, but it is a bit repetitive.

Reply:
I started toying with this a bit; it gets a little more complicated...which is understandable, since we're turning more of this into "magic". I'd love to get this PR merged, then open a draft PR with an approach to this and see if folks would prefer that 🙂